### PR TITLE
Added Netskope Solution

### PR DESCRIPTION
**Pull request by nipun-crestdatasystem**
Change(s):
   - Added Netskope solution

   Version Updated:
   - 3.0.0

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Thanks for raising this PR.
This PR will be investigated and we will come back to you with an update by 09 April, 2024

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Does this Data connector  " Netskope WebTransactions Data Connector " uses any zip file to deploy function app

**Comment by nipun-crestdatasystem** [comment]
@v-prasadboke no this does not use any zip file to deploy the function app.

File: Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json
Comment (v-prasadboke) [inline Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json:1]: Hello @nipun-crestdatasystem, 
Can you provide a readme file for this Data connector

File: Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/azuredeploy_Connector_NetskopeWebTransaction.json [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+            "properties": {
+                "publisher": "Microsoft.Azure.Extensions",
+                "type": "CustomScript",
+                "typeHandlerVersion": "2.1",
+                "autoUpgradeMinorVersion": true,
+                "settings": {},
+                "protectedSettings": {
+					"commandToExecute": "[concat('sudo sh install_docker.sh ',parameters('DockerImageName'),' ',parameters('NetskopeHostName'),' ',parameters('NetskopeAPIToken'),' ',parameters('WorkspaceKey'),' ',parameters('WorkspaceID'),' ',parameters('BackoffRetryCount'),' ',parameters('BackoffSleepTime'),' ',parameters('IdleTimeout'),' ',parameters('SeekTimestamp'))]",
+                    "fileUris": [
+                        "https://aka.ms/sentinel-InstallDocker"
```
Comment (v-prasadboke) [inline Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/azuredeploy_Connector_NetskopeWebTransaction.json:380]: Also I want actual hyperlink to this shortlink url

Comment (nipun-crestdatasystem) [reply Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/azuredeploy_Connector_NetskopeWebTransaction.json:380]: Raw URL for file :   Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/install_docker.sh

Comment (v-prasadboke) [reply Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/azuredeploy_Connector_NetskopeWebTransaction.json:380]: Thanks

File: Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json
Comment (nipun-crestdatasystem) [reply Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json:1]: @v-prasadboke Here a specific readme is not required, the steps are already provided on the data connector page Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json
Let me know if you have any questions.
Thanks for your patience.

Comment (v-prasadboke) [reply Solutions/Netskopev2/Data Connectors/NetskopeWebTransactionsDataConnector/Netskope_WebTransactions.json:1]: Thanks for the response @nipun-crestdatasystem

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, There are 2 zips in the data connector folder. Which one is the function app for Netskope Data connector

**Comment by nipun-crestdatasystem** [comment]
> Hello @nipun-crestdatasystem, There are 2 zips in the data connector folder. Which one is the function app for Netskope Data connector

@v-prasadboke 
The Netskope Data Connector itself consists of 3 Azure Function Apps, so we have 3 zips for the same.
So we need to map:
1. https://aka.ms/sentinel-NetskopeToStorage-functionapp -> Solutions/Netskopev2/Data Connectors/NetskopeDataConnector/NetskopeToStorage.zip
2. https://aka.ms/sentinel-NetskopeStorageToSentinel-functionapp -> Solutions/Netskopev2/DataConnectors/NetskopeDataConnector/NetskopeStorageToSentinel.zip
3. https://aka.ms/sentinel-NetskopeWebTxMetrics-functionapp -> Solutions/Netskopev2/Data Connectors/NetskopeDataConnector/NetskopeWebTxMetrics.zip

**Comment by nipun-crestdatasystem** [comment]
@v-prasadboke  screenshots of the function app.
image (2)
image (1)
image

Thanks.

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**